app/src/main/java/org/sagebionetworks/research/mpower/studyburst/StudyBurstViewModel.kt

```diff
@@ -0,0 +1,77 @@
+package org.sagebionetworks.research.mpower.studyburst
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import org.sagebionetworks.research.mpower.R
+
+class StudyBurstViewModel : ViewModel() {
+
+    private val title = MutableLiveData<String>()
+    private val message = MutableLiveData<String>()
+    private val expires = MutableLiveData<String>()
+    private var items = MutableLiveData<List<StudyBurstItem>>()
+
+    fun init() {
+        title.setValue("Great start!")
+        message.setValue("Complete the rest of todays activities within the next hour in order to complete the study burst for Day 12.")
+        expires.setValue("00:53:04")
+
+        // TODO: remove when we have real data
+        val list = arrayListOf<StudyBurstItem>()
```

Comment:
I'm working on the data model right now.  I should be able to easily integrate with this ViewModel class.